testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -147,6 +155,20 @@ private DistributedQueryRunner(
                     baseDataDir,
                     systemAccessControls));
             servers.add(coordinator);
+            if (nodeCount >= 2 && backupCoordinatorProperties.isPresent()) {
```

Comment:
add `checkArgument(nodeCount >2, ...` in the `backupCoordinatorProperties.isPresent()` section in the constructor and drop it from this if